apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx

```diff
@@ -329,6 +335,7 @@ const PlanUpdateSidePanel = () => {
         subscription={subscription}
         onClose={() => setSelectedTier(undefined)}
         onConfirm={onConfirmDowngrade}
+        projects={orgProjects}
```

Comment:
is DowngradeModal using this? i might be missing something sorry but i don't see any updates to DowngradeModal in the PR, and it doesnt have the projects props on develop too

Reply:
Picked a wrong change, removed it